katalog/prometheus-adapter/apiService.yaml

```diff
@@ -1,21 +1,22 @@
-# Copyright (c) 2020 SIGHUP s.r.l All rights reserved.
+# Copyright (c) 2022 SIGHUP s.r.l All rights reserved.
 # Use of this source code is governed by a BSD-style
 # license that can be found in the LICENSE file.
 
----
 apiVersion: apiregistration.k8s.io/v1
 kind: APIService
 metadata:
-  name: v1beta1.metrics.k8s.io
   labels:
-    app: metrics-server
-  annotations:
-    cert-manager.io/inject-ca-from: "kube-system/metrics-server-tls"
+    app.kubernetes.io/component: metrics-adapter
+    app.kubernetes.io/name: prometheus-adapter
+    app.kubernetes.io/part-of: kube-prometheus
+    app.kubernetes.io/version: 0.9.1
+  name: v1beta1.metrics.k8s.io
 spec:
   group: metrics.k8s.io
   groupPriorityMinimum: 100
-  versionPriority: 100
+  insecureSkipTLSVerify: true
```

Comment:
It is from upstream. The only way to avoid it would to introduce the `cert-manager` package as a dependency for this module but I don't think it's worth it.